README.md

```diff
@@ -44,6 +49,14 @@ To scaffold a new plugin inside this repo run the following command:
 yarn run create:plugin "my plugin"
 ```
 
+## Contributing
+
+Feel free to open an [issue](https://github.com/intuit/design-systems-cli/issues) or a [pull request](https://github.com/intuit/design-systems-cli/pulls)!
+
+Make sure to read our [code of conduct](./CODE_OF_CONDUCT.md).
+
+We actively welcome pull requests. Learn how to [contribute](./CONTRIBUTING.md).
```

Comment:
The link to `CONTRIBUTING.md` will not be broken after https://github.com/intuit/design-systems-cli/issues/531 issue gets resolved.